### ISSUE TITLE
Classic environments in separate package?

### ISSUE THREAD

**Issue by darsnack** [JuliaReinforcementLearning/ReinforcementLearningEnvironments.jl#73]
Opening this up because of https://github.com/JuliaReinforcementLearning/CommonRLInterface.jl/issues/18#issuecomment-647616237. This package has some classic environments implemented, as well as a lot of wrapped environments. In that sense, this package does function as a "glue" package (a one-stop shop as mentioned in the README).

As discussed in the linked comment, there are some features that could be added to the classic environments here. I was considering doing that as a PR, but then I thought it might make more sense to split that into its own package. I could do that in FluxML/Gym.jl (I think we can also take ownership of this repo if preferred), since it already supported the rendering logic which is the most involved part. Then this package could just be a "glue"/wrapper package.

**Comment by jbrea** [comment]
In fact, we had these in a [seperate package](https://github.com/JuliaReinforcementLearning/ReinforcementLearningEnvironmentClassicControl.jl) some time ago... I think since ReinforcementLearningEnvironments.jl is pretty light-weight without the optionally loaded packages, @findmyway decided to integrate the classic control environments.
I am fine with both options, seperating them out or keeping them here...

**Comment by findmyway** [comment]
That's ok to me. My only concern is, that repo seems not to be actively maintained.

Another issue is, both packages have some AbstractSpace implemented. I'm not sure if they are same or not. But we'd better only have one. Or we can wait for the next release of CommonRLInterface to have some decisions on this aspect.

**Comment by darsnack** [comment]
Yeah if I were to go that route, I would throw about the abstractions in that package and use one of the RL base packages in either this org, JuliaPOMDPs, or CommonRLInterface.jl.
